e2e/events.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { test, expect } from '@playwright/test';
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('http://localhost:3000');
+});
+
+
+test.describe('Test if see more link is clickable', () => {
+  test('should show react dev community page', async ({ page }) => {
+    test.slow();
+    const linkName = await page.locator('div.see-more-div')
+    await linkName.click();    
+    const title = await page.locator('div.name').nth(1)
+    await expect(title).toBeTruthy();
+  });
+});
+
+
+test.describe('Test if see more link exists', () => {
+  test('should show See More', async ({ page }) => {
+    await page.waitForNavigation();
+    const linkName = page.locator('div.see-more-div');
+    await expect(linkName).toBeTruthy();
+
+  });
+});
```

Comment:
These tests do not seem to be testing what we are after in the requirements. Also some pointers: there is no need to mark the test as `slow` - it is not doing any long operation, it is advisable to avoid selectors tied to implementation like `nth` https://playwright.dev/docs/selectors#best-practices, always grouping individual tests in `describe` is usually unnecessary.

This is more of what we intended to test:

```ts
test('should navigate to the correct URL when See More is clicked', async ({
  page,
}) => {
  const [newPage] = await Promise.all([
    page.waitForEvent('popup'),
    page.locator('#events >> text="See More"').click(),
  ]);
  await expect(newPage).toHaveURL(
    'https://kommunity.com/reactjs-developer-community-kenya-reactdevske/events'
  );
});
```

Reply:
the test you've given is perfect, it works well for chrome, firefox but not safari.
Safari needs to scroll the page until the #events >> text="See More' is in view before clicking it, otherwise it will fail.
The solution for it is adding **test.slow()** after  `page.locator('#events >> text="See More"').click(),` for it to work universally across all browsers factoring in safari


>const [newPage] = await Promise.all([
    page.waitForEvent('popup'),
    page.locator('#events >> text="See More"').click(),
   test.slow()
  ]);

If you're not experiencing this issue and all tests are passing in all the 3 browsers,, just ignore this.

Reply:
The test passed in all the 3 browsers in CI.